### DISCUSSION TITLE
Are there additional styling options?

### DISCUSSION THREAD

**Discussion by joecaraccio** [payloadcms/payload#715] [Feature Requests & Ideas]
Hey Everyone,

Love Payload so far!

My only one complaint is while the Ui is elegant, I wish there were other options. Such as a compressed User Interface so it plays a little nicer on mobile screens.

I thought about extending some of the styling. I notici3d from the docs that it seems that you have very little control with those variable overrides.

Are there any other options to restyle Payload? Other than just keeping the source locally and editing?

**Comment by jmikrut** [comment]
Hey @joecaraccio — I believe you have already found how to customize CSS, which within, you can control 100% of the CSS. We are going to respond to #716 shortly with follow-up questions for you because that should work.

But, outside of this, we actually agree with you in that we could be a bit more "compressed" and especially nicer on mobile. This is actually planned for our 1.0 release, and things will be changing / improving here very, very shortly.

You'll see more from us regarding mobile UX and better use of space shortly!
